Added recipes, attributes, and tests for remi-php70.

ikari7789 opened the pull request:
### Summary
Bumped version to 2.0.0.
Added missing repositories available from remi.
Updated repository URLs.
Updated tests.

### Description
I've added attributes and recipes for the current remi repository setup. Since it changes the URLs for the repositories to match the currently used URL, I consider this a breaking change, and I have bumped the version to 2.0. Besides that, I also added conflicting checks for the various repositories as they cannot exist in harmony with each other. This is my first time really working with Chef, so if there is anywhere for improvement, I will fix it.

iennae commented:
Hey @ikari7789, As I was reviewing this I found failures with converging some of the fedora images. These changes also increase the number of suites to be done. Any way you can simplify the test matrix so it doesn't take up so much more time? Please also rebase your commits based on current head. In general don't change the version, just signal that it needs to be a breaking change if necessary. Really appreciate your contributions, and look forward to your updates. Thanks!

tas50 commented:
@ikari7789 Can you rebase this so I can take a swing at merging it in?

ikari7789 commented:
Sorry for the delay on this. I've rebased and letting the tests run now. @iennae do you have any suggestions how to simplify the test matrix? I removed the "-test" recipes from the test matrix for now, which should speed things up a bit.

iennae commented:
Thanks for the update @ikari7789. I'll check up on this tomorrow. Thanks so much!

ikari7789 commented:
This PR is too massive and too difficult to review. I'm going to make a series of smaller PRs to replace this one.

iennae commented:
Thanks @ikari7789 appreciate it!
